Sources/ValCommand/ValCommand.swift

```diff
@@ -57,7 +57,16 @@ public struct ValCommand: ParsableCommand {
   /// The default location of Val's SDK.
   private static func defaultValSDK() -> URL {
     #if os(Windows)
-      return URL(fileURLWithPath: FileManager.default.currentDirectoryPath, isDirectory: true)
+      let environment = ProcessInfo.processInfo.environment["USERPROFILE"]!
+      //For CI
+      if environment.contains("runneradmin") {
+        return URL(
+          fileURLWithPath:
+            "D:/a/val/val/.build/x86_64-unknown-windows-msvc/release/Support.build/src/.val")
```

Comment:
```suggestion
            "D:/a/val/val/.build/x86_64-unknown-windows-msvc/release/Support.build/src/.val")
```
There's got to be another way than hardcoding paths in the source. Why not set an environment variable when you configure the CI?

Reply:
The reason we need to do these thing in CI is because that Github Action does not seem to allow direct access to the user's directory.

Reply:
I understand. I'm suggesting that you read an environment variable that gets set by the CI configuration script before the tests are ran. You can hardcode this path in the environment variable.

I just don't want to have hardcoded paths in the source files.